### PR TITLE
#171927127 add some unit tests to hit our target of 96%

### PR DESCRIPTION
**Pull request by hakoemmy**
### What does this PR do?

-  Test uncovered lines in ```validateTrip.js``` middleware

### How should this be manually tested?

- Clone the repo via ```git clone <repo-link>```
- Launch the terminal 
- Run ```npm run test``` 
- Then use ```nyc reporter``` to see if all lines in  ```validateTrip.js``` middleware are covered

### Implementation plan
N/A
### Stories associated with PR

[#171927127](https://www.pivotaltracker.com/story/show/171927127)

### Screenshots if any
Screen Shot 2020-03-23 at 18 45 31

### CODE REVIEW

File: src/tests/mock/editTripRequestMock.js [hunk 1]
```diff
@@ -13,4 +14,133 @@ export const updatedTripRequest = {
...
+	departureDate:'2020-07-17',
+	accommodation:'no',
+	returnDate:'2020-08-28'
+};
+const socketURL = 'http://localhost:3000';   
+const options = {
+  transports: ['websocket'],
+  'force new connection': true
+};
+export const client = io.connect(socketURL, options);
```
Comment (AbdoulNuru) [inline src/tests/mock/editTripRequestMock.js]: @hakoemmy  Add a new line

Comment (AbdoulNuru) [reply src/tests/mock/editTripRequestMock.js]: And did you test those lines too?

Screen Shot 2020-03-23 at 19 21 30

**Comment by blaise82** [comment]
Screen Shot 2020-03-23 at 19 41 07

for more https://travis-ci.org/github/andela/octopus-bn-backend/builds/665966679?utm_source=github_status&utm_medium=notification

from the look of it, we still got a lot of uncovered lines please take your time and try to cover them if possible.

**Review by RusimbiPatrick — COMMENTED** [review]
You set the target too high and you did not achieve it

File: src/tests/mock/editTripRequestMock.js (hunk 1 above)
Comment (hakoemmy) [reply src/tests/mock/editTripRequestMock.js]: @AbdoulNuru Thanks, I've already tested them like so

**Comment by hakoemmy** [comment]
> Screen Shot 2020-03-23 at 19 41 07
> 
> for more https://travis-ci.org/github/andela/octopus-bn-backend/builds/665966679?utm_source=github_status&utm_medium=notification
> 
> from the look of it, we still got a lot of uncovered lines please take your time and try to cover them if possible.

> My target was to test the uncovered lines in ```validateTrip``` middleware so I got them tested like so

**Comment by hakoemmy** [comment]
> You set the target too high and you did not achieve it

> Thanks for the correction, I messed up this with our target at front-end, I changed the target to 95%

**Review by AbdoulNuru — APPROVED** [review]

**Review by RusimbiPatrick — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Pull request merged**